gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -62,17 +64,85 @@
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
   private static final String GCD = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
Lets make GCD be based on GCD_VERSION (GCD_VERSION + "...").
Feel free to find a better name for "GCD" and consider a constant for build number ("rev1-2.1.2.b").